game_patch/multi/server.cpp

```diff
@@ -327,29 +327,24 @@ const char* get_rand_level_filename()
     std::size_t num_levels = rf::netgame.levels.size();
 
     // current_level_index keeps our position in rotation while we go somewhere else
-    const char* current_level_filename = rf::level_filename_to_load;
-
-    xlog::debug("Starting random selection process, current level is {}", current_level_filename);
+    rf::String filename = rf::level_filename_to_load;
```

Comment:
```suggestion
    const rf::String& filename = rf::level_filename_to_load;
```
or maybe
```suggestion
   const std::string_view filename= rf::level_filename_to_load;
```

Reply:
Is there a benefit to this? I don't mean to come across as arrogant in saying this, I just legitimately do not see a benefit of this - could you explain?

Reply:
To avoid a copy involving a heap allocation.

Reply:
I initially accepted this change, however then realized when attempting to build that it would make me unable to set `filename` to `rf::netgame.levels[rand_level_index]` further down.